.github/autobump.txt

```diff
@@ -1654,6 +1676,7 @@ xplr
 xray
 xrootd
 xterm
+xz
```

Comment:
Should we be autobumping formulae to that require additional work (uploading an archive)?

Reply:
That's not the primary download right? Then I think it's fine. But automatically doing that would be even better.